tests/models/test_model.py

```diff
@@ -325,3 +372,21 @@ def test_model_uuid():
     m_dict.pop("model_uuid")
     m4 = Model.from_dict(m_dict)
     assert m4.model_uuid is None
+
+
+def test_validate_schema(sklearn_knn_model, iris_data, tmpdir):
+    sk_model_path = os.path.join(str(tmpdir), "sk_model")
+    X, y = iris_data
+    signature = infer_signature(X, y)
+    with mlflow.start_run():
+        mlflow.sklearn.save_model(
+            sklearn_knn_model,
+            sk_model_path,
+            signature=signature,
+        )
```

Comment:
```suggestion
    mlflow.sklearn.save_model(
        sklearn_knn_model,
        sk_model_path,
        signature=signature,
    )
```

My code suggestion contained `start_run`, but we don't need it.